triggers/daml/Daml/Trigger/LowLevel.daml

```diff
@@ -155,8 +166,8 @@ data ActiveContracts = ActiveContracts { activeContracts : [Created] }
 -- | Trigger is (approximately) a left-fold over `Message` with
 -- an accumulator of type `s`.
 data Trigger s = Trigger
-  { initialState : Party -> Time -> ActiveContracts -> (s, [Commands])
-  , update : Time -> Message -> s -> (s, [Commands])
+  { initialState : Party -> ActiveContracts -> Free TriggerF s
```

Comment:
While this is only the LowLevel interface, some newtype around `Free TriggerF s` would probably still be nice so that `submitCommands` can also be used here if you really want this interface with all the caveats that come with it.

Reply:
It's already possible to do this by writing `runTriggerSetup do`.

Reply:
good point, you still end up leaking `Free` into the public API though which I’d prefer to avoid.

Reply:
I was avoiding this because (as you mentioned in another comment) newtypes add layers, which is annoying to deal with at the Scala layer as seen in 841ee75320120e2300d34bf2ec563e17299029af .  Anyway that is there now.